client/protractor-conf.js

```diff
@@ -25,7 +25,7 @@ exports.config = {
     framework: 'jasmine2',
     jasmineNodeOpts: {
         showColors: true,
-        defaultTimeoutInterval: 55000
+        defaultTimeoutInterval: 120000
```

Comment:
This is the time a test can run and because we have now larger tests suits, more time is needed.